python/src/internal/errors.hh

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+#include <Python.h>
+
+namespace nix::python {
+
+extern PyObject * NixError;
+extern PyObject * ThrownNixError;
+}
```

Comment:
I think it would make sense to expose the headers, so that an application can extend the bindings if necessary. Perhaps they want to provide bindings for a Nix plugin?
This is a very advanced use case, and considering that we don't provide any guarantees about headers anyway, I think it's fine.